### DISCUSSION TITLE
quick navigation through search window

### DISCUSSION THREAD

**Discussion by loulou91** [photoprism/photoprism#4135] [Q&A]
By default search window displays thumbnails of all pictures (50k in my case). Navigation through this large amount is possible using mouse scrolling and lift on the right side of the window (windows 11 & Chrome).
Of course when you have a large amount of pictures scrolling is tedious and scrolling inside the lift as well. In fact, lift looks to cover not all pictures from the first one to the last one. I have to scroll many many times to go to the end!
Is there any tip to go directly and quickly to any place in the list of picture? The only one I saw (in addition to filters of course) is an arrow to go back to the top of the window.
As an example, Google photos presents a calendar at the right allowing direct access to any period.

Thanks in advance.

**Comment by loulou91** [comment]
No reaction of the community on this issue? Did I missed something?

**Comment by graciousgrey** [comment]
A timeline view is planned for the future. Until it is implemented you can refine the search by using additional search filters (e.g. year, month, before, after) or by sorting by newest or oldest first.

- https://github.com/photoprism/photoprism/issues/152

**Reply by loulou91** [reply]
Noted, thanks.
